robot/impl/local_robot_test.go

```diff
@@ -3027,6 +3027,110 @@ func TestCrashedModuleReconfigure(t *testing.T) {
 	})
 }
 
+func TestModularResourceReconfigurationCount(t *testing.T) {
```

Comment:
can you add a test for module + resource reconfiguration at the same time?

Reply:
Yep! It wasn't there before; added.

Reply:
Also added a service to the existing tests to make sure those code paths are correct.